src/Plugin/tests/VoteUpDownWidgetManagerTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\Tests\vud\Functional;
```

Comment:
I think you want other namespace, please read [documentation about unit testing](https://api.drupal.org/api/drupal/core!core.api.php/group/testing/8.3.x).